public/app/features/plugins/sql/components/configuration/ConnectionLimits.tsx

```diff
@@ -81,67 +90,140 @@ export const ConnectionLimits = <T extends SQLConnectionLimits>(props: Props<T>)
     });
   };
 
+  const labelWidth = 40;
+
   return (
-    <FieldSet label="Connection limits">
-      <InlineField
-        tooltip={
-          <span>
-            The maximum number of open connections to the database.If <i>Max idle connections</i> is greater than 0 and
-            the <i>Max open connections</i> is less than <i>Max idle connections</i>, then
-            <i>Max idle connections</i> will be reduced to match the <i>Max open connections</i> limit. If set to 0,
-            there is no limit on the number of open connections.
-          </span>
+    <ConfigSection title="Connection limits">
```

Comment:
I think this should be `ConfigSubSection` instead of `ConfigSection`. If you take a look at MSSQL we're nesting this component in the additional settings sub-section. Wdyt?

Reply:
pinging @gwdawson 😄

Reply:
yeah, i think this makes sense.. ill update this to be `ConfigSubSection`.